Traceback (most recent call last):
  File "C:\Users\yanin\anaconda3\lib\site-packages\jupyter_cache\executors\utils.py", line 58, in single_nb_execution
    executenb(
  File "C:\Users\yanin\anaconda3\lib\site-packages\nbclient\client.py", line 1204, in execute
    return NotebookClient(nb=nb, resources=resources, km=km, **kwargs).execute()
  File "C:\Users\yanin\anaconda3\lib\site-packages\nbclient\util.py", line 84, in wrapped
    return just_run(coro(*args, **kwargs))
  File "C:\Users\yanin\anaconda3\lib\site-packages\nbclient\util.py", line 62, in just_run
    return loop.run_until_complete(coro)
  File "C:\Users\yanin\anaconda3\lib\asyncio\base_events.py", line 649, in run_until_complete
    return future.result()
  File "C:\Users\yanin\anaconda3\lib\site-packages\nbclient\client.py", line 663, in async_execute
    await self.async_execute_cell(
  File "C:\Users\yanin\anaconda3\lib\site-packages\nbclient\client.py", line 965, in async_execute_cell
    await self._check_raise_for_error(cell, cell_index, exec_reply)
  File "C:\Users\yanin\anaconda3\lib\site-packages\nbclient\client.py", line 862, in _check_raise_for_error
    raise CellExecutionError.from_cell_and_msg(cell, exec_reply_content)
nbclient.exceptions.CellExecutionError: An error occurred while executing the following cell:
------------------
fixedpt(lambda x: x-(x**2-2)/4, -5, 1e-5, 15)
------------------

[1;31m---------------------------------------------------------------------------[0m
[1;31mOverflowError[0m                             Traceback (most recent call last)
Cell [1;32mIn[7], line 1[0m
[1;32m----> 1[0m [43mfixedpt[49m[43m([49m[38;5;28;43;01mlambda[39;49;00m[43m [49m[43mx[49m[43m:[49m[43m [49m[43mx[49m[38;5;241;43m-[39;49m[43m([49m[43mx[49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[38;5;241;43m2[39;49m[38;5;241;43m-[39;49m[38;5;241;43m2[39;49m[43m)[49m[38;5;241;43m/[39;49m[38;5;241;43m4[39;49m[43m,[49m[43m [49m[38;5;241;43m-[39;49m[38;5;241;43m5[39;49m[43m,[49m[43m [49m[38;5;241;43m1e-5[39;49m[43m,[49m[43m [49m[38;5;241;43m15[39;49m[43m)[49m

Cell [1;32mIn[1], line 6[0m, in [0;36mfixedpt[1;34m(g, pzero, eps, N)[0m
[0;32m      4[0m [38;5;28;01mwhile[39;00m n[38;5;241m<[39mN:
[0;32m      5[0m     [38;5;28mprint[39m(n)
[1;32m----> 6[0m     pone [38;5;241m=[39m [43mg[49m[43m([49m[43mpzero[49m[43m)[49m
[0;32m      7[0m     [38;5;28;01mif[39;00m np[38;5;241m.[39mabs(pone[38;5;241m-[39mpzero)[38;5;241m<[39meps:
[0;32m      8[0m         [38;5;28mprint[39m([38;5;124m'[39m[38;5;124mp is [39m[38;5;124m'[39m, pone, [38;5;124m'[39m[38;5;124m and the iteration number is [39m[38;5;124m'[39m, n)

Cell [1;32mIn[7], line 1[0m, in [0;36m<lambda>[1;34m(x)[0m
[1;32m----> 1[0m fixedpt([38;5;28;01mlambda[39;00m x: x[38;5;241m-[39m([43mx[49m[38;5;241;43m*[39;49m[38;5;241;43m*[39;49m[38;5;241;43m2[39;49m[38;5;241m-[39m[38;5;241m2[39m)[38;5;241m/[39m[38;5;241m4[39m, [38;5;241m-[39m[38;5;241m5[39m, [38;5;241m1e-5[39m, [38;5;241m15[39m)

[1;31mOverflowError[0m: (34, 'Result too large')
OverflowError: (34, 'Result too large')

